pkgs/development/web/playwright/driver.nix

```diff
@@ -161,13 +161,13 @@ let
       browsersJSON = (lib.importJSON ./browsers.json).browsers;
       browsers =
         {
-          x86_64-linux = browsers-linux { };
-          aarch64-linux = browsers-linux { };
-          x86_64-darwin = browsers-mac;
-          aarch64-darwin = browsers-mac;
+          x86_64-linux = browsers { };
+          aarch64-linux = browsers { };
+          x86_64-darwin = browsers { withWebkit = false; };
```

Comment:
Is there something blocking webkit on darwin? If so, we should probably document that at least in a comment or so. Otherwise: lets enable it ;)

Reply:
Mac OS has downloads for each os version, eg. mac-14-arm, mac-15-arm etc, I'm not sure from which archive should we provide webkit in nixpkgs?

Reply:
enabled using mac-14 build, works on my macos 15 as well